pytestqt/plugin.py

```diff
@@ -709,7 +721,14 @@ def _handle(self, msg_type, message):
                 ignored = True
                 break
 
-        self._records.append(Record(msg_type, message, ignored))
+        if context is not None:
+            context = self._Context(
+                to_unicode(context.file),
+                to_unicode(context.function),
+                context.line,
```

Comment:
Note `context.file` and `context.function` can also be `None` in rare circumstances. I don't really know when it happens, but I have seen it happen (and handle it in my custom message handler).